toolchain/gcc/common.mk

```diff
@@ -188,8 +192,6 @@ ifneq ($(GCC_PREPARE),)
 	$(CP) $(SCRIPT_DIR)/config.{guess,sub} $(HOST_SOURCE_DIR)/
 	$(SED) 's,^MULTILIB_OSDIRNAMES,# MULTILIB_OSDIRNAMES,' $(HOST_SOURCE_DIR)/gcc/config/*/t-*
 	$(SED) 'd' $(HOST_SOURCE_DIR)/gcc/DEV-PHASE
-	$(SED) 's, DATESTAMP,,' $(HOST_SOURCE_DIR)/gcc/version.c
```

Comment:
Why this change?

want to cherry pick my fix ?

https://github.com/openwrt/openwrt/pull/9852

Reply:
8-11
```patch
--- a/gcc/version.c
+++ b/gcc/version.c
@@ -28,8 +28,8 @@ along with GCC; see the file COPYING3.
 const char bug_report_url[] = BUGURL;
 
 /* The complete version string, assembled from several pieces.
-   BASEVER, DATESTAMP, DEVPHASE, and REVISION are defined by the
+   BASEVER, DEVPHASE, and REVISION are defined by the
    Makefile.  */
 
-const char version_string[] = BASEVER DATESTAMP DEVPHASE REVISION;
+const char version_string[] = BASEVER DEVPHASE REVISION;
 const char pkgversion_string[] = PKGVERSION;
```
12
```patch
--- a/gcc/genversion.cc
+++ b/gcc/genversion.cc
@@ -67,10 +67,10 @@ main (void)
   printf ("#define GCC_major_version %lu\n\n", major);
 
   printf ("/* The complete version string, assembled from several pieces.\n"
-	  "BASEVER, DATESTAMP, DEVPHASE, and REVISION are defined by the\n"
+	  "BASEVER, DEVPHASE, and REVISION are defined by the\n"
 	  "Makefile.  */\n\n");
 
-  printf ("#define version_string \"" BASEVER DATESTAMP DEVPHASE REVISION "\"\n");
+  printf ("#define version_string \"" BASEVER DEVPHASE REVISION "\"\n");
   printf ("#define pkgversion_string \"" PKGVERSION "\"\n\n");
 
   printf ("/* This is the location of the online document giving instructions for\n"
```

Reply:
?

Reply:
A suggest solution. To create a patch to drop DATESTAMP from displayed version.

Reply:
Mhe an additional patch? We can do that with 2 additional line in the makefile

Reply:
The first commit(https://github.com/openwrt/openwrt/commit/6ac26a1ef71c0d7a0d38572f6bc0698be11616bf) about that was made by @nbd168.
I'm not sure which solution is the best now.

Reply:
> Why this change?

Just to get it to build. It's harmless in itself, but it does affect reproducibility of the builds, that's probably why it was done in the first place.

> want to cherry pick my fix ?

Thanks, I had solved it locally in an equivalent way (mostly a style difference, but it's my personal preference). Will push soon.